test/docs/__snapshots__/get-docs-yaml.api-test.ts.snap

```diff
@@ -91,6 +91,11 @@ info:
 tags: []
 servers: []
 components:
+  securitySchemes:
+    ApiKeyHeader:
+      type: apiKey
+      in: header
+      name: x-api-key
```

Comment:
Depending on how we have clients authenticate with the gateway, this might need to change slightly for the deployed swagger spec (e.g. if the gateway uses a different header for the API key). I think we can worry about that when it becomes a problem, rather than in advance